src/main/groovy/com/ullink/Msbuild.groovy

```diff
@@ -127,9 +127,21 @@ class Msbuild extends ConventionTask {
                     parseProject = false
                 } else {
                     projectParsed = allProjects[projectName]
```

Comment:
Why would one access a project that does not exist and why would we try to guess why she should actually access instead